src/Octoshift/Services/GithubApi.cs

```diff
@@ -722,6 +722,25 @@ public virtual async Task<IEnumerable<Mannequin>> GetMannequins(string orgId)
         }
     }
 
+    public virtual async Task<IEnumerable<Mannequin>> GetMannequinsByLogin(string orgId, string login)
```

Comment:
Nitpick: if we're just getting one mannequin, should we change the name to `GetMannequinByLogin`?

Reply:
I think the plural is right - see my [other comment](https://github.com/github/gh-gei/pull/1067#discussion_r1268309196).